openpype/hosts/celaction/plugins/publish/collect_celaction_cli_kwargs.py

```diff
@@ -9,15 +10,18 @@ class CollectCelactionCliKwargs(pyblish.api.Collector):
     order = pyblish.api.Collector.order - 0.1
 
     def process(self, context):
-        kwargs = celaction.kwargs.copy()
+        passing_kwargs = scripts.PASSING_KWARGS.copy()
```

Comment:
You can actually pass the content of `cli` function right here to avoid the `PASSING_KWARGS` variable (which is not nice to have there).
```suggestion
        parser = argparse.ArgumentParser(prog="celaction_publish")
        parser.add_argument("--currentFile",
                            help="Pass file to Context as `currentFile`")
        parser.add_argument("--chunk",
                            help=("Render chanks on farm"))
        parser.add_argument("--frameStart",
                            help=("Start of frame range"))
        parser.add_argument("--frameEnd",
                            help=("End of frame range"))
        parser.add_argument("--resolutionWidth",
                            help=("Width of resolution"))
        parser.add_argument("--resolutionHeight",
                            help=("Height of resolution"))
        passing_kwargs = parser.parse_args(sys.argv[1:]).__dict__
```